crates/pop-parachains/src/call/metadata/params.rs

```diff
@@ -20,6 +20,8 @@ pub struct Param {
 	pub is_tuple: bool,
 	/// Indicates if the parameter is a Variant.
 	pub is_variant: bool,
+	/// Indicates if the parameter is a Sequence.
+	pub is_sequence: bool,
```

Comment:
Introduced in https://github.com/r0gue-io/pop-cli/pull/363/commits/f2a0f76f1a4847b5eb294efc2c82c296e05eb697. because it solves the comment to improve tests: https://github.com/r0gue-io/pop-cli/pull/363#discussion_r1875829087 

But I am going to prepare another PR to refactor all the rest of usages.